website/docs/components/input-phone/input-phone-a11y.md

```diff
@@ -4,38 +4,10 @@ a11y: AA
 tabs: Design('input-phone'), A11y('input-phone-a11y'), Example('input-phone-code')
 ---
 
-## What component has
+This component consists of Input and Select elements, so feel free to check their detailed documentation regarding accessibility:
```

Comment:
should the InputMask be mentioned here as well? or we'll do it after we fix its a11y?

Reply:
This is a good point! I wasn't sure should we add since not all the examples use it. But looks like it could be a good addition.